openghg/util/_strings.py

```diff
@@ -170,3 +170,28 @@ def extract_float(s: str) -> float:
         return float(m.group(0))
 
     raise ValueError(f"No float values found in '{s}'")
+
+
+def check_and_set_null_variable(param: Union[str, None], null_value: Optional[str] = None) -> str:
```

Comment:
This is a nice additional. Eventually it should go with the rest of the metadata formatting stuff, but I'll just make a note of that.

Reply:
Yeah agreed - I was struggling to decide where to put it so kept with strings for now but will be better to include somewhere central when we pull together formatting.